sources/packages/web/src/views/supporting-user/Login.vue

```diff
@@ -1,41 +1,42 @@
 <template>
-  <v-card elevation="2" class="mx-auto mt-15" max-width="690px" outlined>
-    <v-card-header>
-      <v-card-header-text>
-        <v-card-title class="my-6 bold-text" style="font-size: 27px"
-          >Welcome to Student Application Supporting Information
-          Portal</v-card-title
-        >
-        <v-card-subtitle>Welcome text goes here…</v-card-subtitle>
-      </v-card-header-text>
-    </v-card-header>
-    <v-card-text
-      >We are using BCSC for authentication. Please click on Login/Register
-      buttons below to start your sign in/sign up.</v-card-text
-    >
-    <v-row class="m-3">
-      <v-col>
-        <v-btn
-          justify-center
-          color="primary"
-          @click="login"
-          class="float-right"
-          prepend-icon="fa:fa fa-user"
-        >
-          Login with BCSC
-        </v-btn>
-      </v-col>
-      <v-col>
-        <v-btn
-          color="primary"
-          @click="login"
-          prepend-icon="mdi-account-star-outline"
-        >
-          Sign Up with BCSC
-        </v-btn>
-      </v-col>
-    </v-row>
-  </v-card>
+  <v-card elevation="2" class="mx-auto mt-15" max-width="730px">
+    <v-card-text>
+      <v-row
+        ><v-col cols="9"
+          ><v-card-header>
```

Comment:
actually, we don't have these old tags  like `v-card-header` in vuetify 3
![image](https://user-images.githubusercontent.com/77353155/195905411-cfecbdda-86b7-4422-9fb3-3e96257208b4.png)

Reply:
https://next.vuetifyjs.com/en/components/cards/#anatomy

Reply:
in other login pages, we are using `<v-card-text>`

Reply:
ok. Done.